### ISSUE TITLE
[Request/Suggestion] Enable System Settings String Change

### ISSUE THREAD

**Issue by rpalmon** [dimok789/mocha#9]
When I want to boot into Mocha I want to know that I am on Mocha. A way you could probably do that is change the string in the system settings that says the version number and make it like [Mocha 5.5.1U]. Something like what you could do with the 3DS. That way I know that I am in the right place and have a lesser chance of bricking my system when I want to install something.

**Comment by CreeperMario** [comment]
I like this idea.
For now, I can differentiate between sysNAND and redNAND by making by folders blue on sysNAND and red on redNAND, but how do I tell if I'm in a sig-patched sysNAND?

This could probably be done by spoofing (or redirecting, in a similar sense to syshax.xml) the version.bin file, unless I'm missing something.

**Comment by dimok789** [comment]
Yeah easy to do but it has some limitations to what can be put in there.
